src/test/java/nl/tudelft/scrumbledore/ConstantsTest.java

```diff
@@ -0,0 +1,36 @@
+package nl.tudelft.scrumbledore;
+
+//import static org.junit.Assert.*;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+
+//import org.junit.Test;
+
+/**
+ * Testing the Constants class.
+ * 
+ * @author Floris Doolaard
+ *
+ */
+public class ConstantsTest {
```

Comment:
I have just added the class just in case some method will arise in the future. The class can be deleted if not so.